packages/core/parcel-bundler/src/builtins/hmr-runtime.js

```diff
@@ -57,7 +59,9 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
     }
 
     if (data.type === 'error') {
-      console.error('[parcel] 🚨  ' + data.error.message + '\n' + data.error.stack);
+      console.error(
+        '[parcel] 🚨  ' + data.error.message + '\n' + (data.error.stack || '')
```

Comment:
It looks like prettier made changes to this file I wasn't expecting. I can get this back to just my changes, but in the meantime, there are only two non-formatting updates to not show `undefined` when the error has no stack.  Here, and on line 89.